endpoints/openrtb2/auction.go

```diff
@@ -296,6 +301,23 @@ func (deps *endpointDeps) validateSite(site *openrtb.Site) error {
 	return nil
 }
 
+func validateUser(user *openrtb.User) error {
+	// DigiTrust support
+	if user != nil && user.Ext != nil {
+		// Creating ExtUser object to check if DigiTrust is valid
+		var userExt openrtb_ext.ExtUser
+		if err := json.Unmarshal(user.Ext, &userExt); err == nil {
```

Comment:
If `err != nil`, shouldn't this also return an error?

That would prevent requests where `user.ext.digitrust` or any of its nested properties are not the right type.

Reply:
yeah, I guess since this is a validate user function we should return an error for that too. added it.